test/test-v8debugapi.js

```diff
@@ -757,10 +759,9 @@ describe('v8debugapi', function() {
           assert.ifError(err);
           var foo = bp.evaluatedExpressions[0];
           var fooVal = bp.variableTable[foo.varTableIndex];
-          assert.equal(fooVal.members.length, 1);
-          assert(foo.status.description.format.indexOf(
-            'Only first') !== -1);
-          assert(!foo.status.isError);
+          // should have 1 element + truncation message.
+          assert.equal(fooVal.members.length, 2);
```

Comment:
Intended. This PR only changes the UX when objects are truncated due to `maxProperties`. The `maxDataSize` will be handled in a follow on change.